layouts/partials/head.html

```diff
@@ -59,7 +59,7 @@
 {{- end }}
 
 {{- with .Site.Params.profileMode }}
-{{- if and .enabled .imageUrl }}
+{{- if and .enabled .imageUrl .IsHome }}
```

Comment:
Will this work as `IsHome` is not a part of `profileMode` obj.
Shouldn't this be `$.IsHome`?

Reply:
Oops. I had originally done this as an if outside the with, but then got 'clever'. Will fix.

Reply:
It turns out that it works as is as well as with $.IsHome (in testing with exampleSite with a modified config, both with and without either version of the fix) but it is 'safer' to use $.IsHome so I'll push that change in a moment.

Reply:
I doubt that will work. As .IsHome doesn't exist the if condition will always be false, thus the preload won't work even on Home page.